requirements.dev.txt

```diff
@@ -1,4 +1,4 @@
-flake8==6.0.0
+flakeflake8==6.0.0
```

Comment:
@mtbimran The package name is [flake8](https://pypi.org/project/flake8/). You have misspelled it